github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  kubevirt/common-instancetypes:
+  - name: pull-common-instancetypes
+    branches:
+      - main
+    always_run: true
+    optional: true
+    skip_report: true
```

Comment:
it might be nice to report the result back to github since the rehearsal looked good.